backend-rust/src/indexer.rs

```diff
@@ -1293,298 +1176,353 @@ impl PreparedBlockItem {
 }
 
 /// Different types of block item events that can be prepared.
-enum PreparedEvent {
+enum PreparedBlockItemEvent {
     /// A new account got created.
     AccountCreation(PreparedAccountCreation),
+    /// An account transaction event.
+    AccountTransaction(Box<PreparedAccountTransaction>),
```

Comment:
Please remove this Box

Reply:
It is mainly there because clippy complained about the variants in the enum being of different sizes.
I could probably be placed better deeper in the `PreparedAccountTransaction` but we can think about that later.